### DISCUSSION TITLE
更新后#Tags不可用了。

### DISCUSSION THREAD

**Discussion by JingWucc** [MuiseDestiny/zotero-style#277] [Q&A]
我最喜欢这个#Tages这个功能了，这样每个文献的重点我一目了然。现在更新后#Tages是空白
希望大佬能够修复一下### 
![image](https://user-images.githubusercontent.com/126381607/225937663-e761e17a-36f2-4257-b076-1ac555d4f0ae.png)

**Comment by MuiseDestiny** [comment]
邮件列设置Match删除#，因为我改变了默认设置，需要手动更改一下

**Reply by MuiseDestiny** [reply]
![image](https://user-images.githubusercontent.com/51939531/225939421-8bb9d72b-ae54-4e27-9fcb-bdc6b3ea9a67.png)

**Reply by JingWucc** [reply]
好的谢谢啦~ 可以了
